tests/p2p/test_capabilities.py

```diff
@@ -5,10 +5,7 @@
 settings = HathorSettings()
 
 
-class HathorCapabilitiesTestCase(unittest.TestCase):
-    def setUp(self):
-        super().setUp()
-
+class SyncV1HathorCapabilitiesTestCase(unittest.SyncV1Params, unittest.TestCase):
```

Comment:
Why is it different from the other files?

Reply:
This is test too specific to sync-v1, the next PR should introduce the equivalent test for sync-v2.

Reply:
Ok. Thanks!